### PR TITLE
Fix rebuildstaging

### PR DESCRIPTION
**Pull request by millerdev**
Review by commit 🐡

### CODE REVIEW

File: scripts/rebuildstaging [hunk 1]
```diff
@@ -52,7 +52,7 @@ done
 
 function rebuildstaging() {
     echo "rebuilding staging branch, this might take a while..."
-    python scripts/rebuildstaging.py < scripts/staging.yaml "$@"
+    python scripts/rebuildstaging.py scripts/staging.yaml "$@"
```
Comment (orangejenny) [inline scripts/rebuildstaging:55]: Curious, what does this change do?

**Review by orangejenny — APPROVED** [review]

File: scripts/rebuildstaging (hunk 1 above)
Comment (mkangia) [reply scripts/rebuildstaging:55]: looks like its now passing the file path instead of file content.. some shell magic..

Comment (orangejenny) [reply scripts/rebuildstaging:55]: 👍  thanks

Comment (millerdev) [reply scripts/rebuildstaging:55]: Previously it was streaming the file content via stdin, now it's passing the file path. It's hard to use pdb when the process reads stdin.

**Pull request merged**